yarn-project/end-to-end/src/e2e_public_cross_chain_messaging.test.ts

```diff
@@ -187,4 +192,67 @@ describe('e2e_public_cross_chain_messaging', () => {
         .simulate(),
     ).rejects.toThrowError("Invalid Content 'l1_to_l2_message_data.message.content == content'");
   }, 60_000);
+
+  describe('Consuming message from non-portal address', () => {
```

Comment:
The naming here is confusing, you are not consuming from a non-portal, any contract can act as a portal, you are just sending to a portal that is not specified as **your** `portal_address` at deployment.